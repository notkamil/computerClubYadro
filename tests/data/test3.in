2
08:45 18:30
15
08:50 1 client1
09:03 2 client1 1
13:27 4 client1