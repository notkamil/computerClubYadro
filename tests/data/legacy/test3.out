08:45
08:50 1 client1
09:03 2 client1 1
13:27 4 client1
18:30
1 75 04:24
2 0 00:00